pkg/webui/console/containers/application-payload-formatters/downlink.js

```diff
@@ -45,14 +47,18 @@ const m = defineMessages({
   title: 'Default downlink payload formatter',
   infoText:
     'You can use the "Payload formatter" tab of individual end devices to test downlink payload formatters and to define individual payload formatter settings per end device.',
+  downlinkResetWarning:
+    'Viewing the downlink payload formatter is prohibited; overwriting is allowed',
```

Comment:
```suggestion
    'You do not have sufficient rights to view the current downlink payload formatter. Only overwriting is allowed.',
```

Adding the reason for the restriction to increase clarity.